### PR TITLE
test: pin container_linux_version with 1520.6.0 for now

### PR DESCRIPTION
**Pull request by cpanato**
cc @s-urbaniak

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -52,6 +52,11 @@ pipeline {
       defaultValue: '',
       description: 'Hyperkube image. Please define the param like: {hyperkube="<HYPERKUBE_IMAGE>"}'
     )
+    string(
+      name: 'container_linux_version',
+      defaultValue: '1520.6.0',
```
[the remark this answers is not in the document]
Comment (cpanato) [reply Jenkinsfile]: yep! 😄

**Comment by cpanato** [comment]
retest this please

**Comment by s-urbaniak** [comment]
All cloud builds work fine (esp. the blocking azure), hence merging. bm needs investigation and has been restarted by @cpanato

**Comment by squat** [comment]
looks good. @cpanato bare-metal is failing.

**Pull request merged**